hybrid/ddc_invalid.c

```diff
@@ -14,6 +14,9 @@
 int main()
 {
 	pp_cap(cheri_ddc_get());
+	// Before clearing the tag we ensure we have a valid one
+	assert(cheri_address_get(cheri_ddc_get()) == 0 && cheri_tag_get(cheri_ddc_get() == 1));
```

Comment:
`cheri_tag_get(cheri_ddc_get() == 1)` doesn't look right -- should it be `cheri_tag_get(cheri_ddc_get()) == 1` or, perhaps more idiomatically, just `cheri_tag_get(cheri_ddc_get())`?

Reply:
I can't reply to your [comment](https://github.com/capablevms/cheri-examples/pull/39#commitcomment-57175418) below but it is also related to checking the tag. Fix in 9a33253.